crypto test cannot be compiled to native with Camel 4 and Quarkus 3

```
[INFO] [stdout] GraalVM Native Image: Generating 'camel-quarkus-integration-test-crypto-3.0.0-SNAPSHOT-runner' (executable)...
[INFO] [stdout] ========================================================================================================================
[INFO] [stdout] [1/7] Initializing...                                                                                   (10.2s @ 0.32GB)
[INFO] [stdout]  Version info: 'GraalVM 22.3.1.0-Final Java 17 Mandrel Distribution'
[INFO] [stdout]  Java version info: '17.0.6+10'
[INFO] [stdout]  C compiler: gcc (redhat, x86_64, 8.5.0)
[INFO] [stdout]  Garbage collector: Serial GC
[INFO] [stdout]  4 user-specific feature(s)
[INFO] [stdout]  - io.quarkus.runner.Feature: Auto-generated class by Quarkus from the existing extensions
[INFO] [stdout]  - io.quarkus.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [org.jboss.threads] categories
[INFO] [stdout]  - io.quarkus.runtime.graal.ResourcesFeature: Register each line in META-INF/quarkus-native-resources.txt as a resource on Substrate VM
[INFO] [stdout]  - io.quarkus.security.BouncyCastleFeature
[INFO] [stdout] [2/7] Performing analysis...  [*]                                                                       (22.0s @ 3.39GB)
[INFO] [stdout]   14,686 (84.53%) of 17,374 classes reachable
[INFO] [stdout]   17,638 (58.46%) of 30,171 fields reachable
[INFO] [stdout]   59,204 (50.48%) of 117,293 methods reachable
[INFO] [stdout]    1,962 classes,   179 fields, and 7,154 methods registered for reflection
[INFO] [stdout] 
[WARN] [stderr] 2 fatal errors detected:
[WARN] [stderr] com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing org.bouncycastle.pqc.jcajce.provider.rainbow.BCRainbowPublicKey.getEncoded() 
[WARN] [stderr] Parsing context:
[WARN] [stderr] Fatal error:    at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:153)
[WARN] [stderr]    at org.bouncycastle.pqc.jcajce.provider.rainbow.BCRainbowPublicKey.getEncoded(BCRainbowPublicKey.java:166)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:104)
[WARN] [stderr]    at sun.security.pkcs.PKCS8Key.equals(PKCS8Key.java:277)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
[WARN] [stderr]    at java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1026)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
[WARN] [stderr]    at java.util.concurrent.ConcurrentHashMap.putIfAbsent(ConcurrentHashMap.java:1541)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultVirtualInvokeTypeFlow.onObservedUpdate(DefaultVirtualInvokeTypeFlow.java:109)
[WARN] [stderr]    at com.oracle.svm.core.jdk.StringInternSupport.doIntern(StringInternSupport.java:77)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.TypeFlow.update(TypeFlow.java:562)
[WARN] [stderr]    at com.oracle.svm.core.jdk.StringInternSupport.intern(StringInternSupport.java:67)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:488)
[WARN] [stderr]    at com.oracle.svm.core.jdk.Target_java_lang_String.intern(JavaLangSubstitutions.java:170)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
[WARN] [stderr] 
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
[WARN] [stderr] 
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
[WARN] [stderr] Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Error loading a referenced type: java.lang.SecurityException: class "org.bouncycastle.pqc.jcajce.provider.util.KeyUtil"'s signer information does not match signer information of other classes in the same package
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)     at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
[WARN] [stderr]         at parsing org.bouncycastle.pqc.jcajce.provider.rainbow.BCRainbowPublicKey.getEncoded(BCRainbowPublicKey.java:169)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:97)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2518)
[WARN] [stderr]         at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:84)
[WARN] [stderr]         at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:110)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3393)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:446)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3345)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3190)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:685)
[WARN] [stderr] Caused by: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Error loading a referenced type: java.lang.SecurityException: class "org.bouncycastle.pqc.jcajce.provider.util.KeyUtil"'s signer information does not match signer information of other classes in the same package
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:171)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.loadReferencedType(WrappedConstantPool.java:163)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:349)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.loadReferencedType(WrappedConstantPool.java:169)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
[WARN] [stderr]         at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SubstrateClassInitializationPlugin.loadReferencedType(SubstrateClassInitializationPlugin.java:58)
[WARN] [stderr]         ... 13 more
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.loadReferenceType(BytecodeParser.java:4290)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5288)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.maybeEagerlyResolve(BytecodeParser.java:4272)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3385)
[WARN] [stderr]         at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.maybeEagerlyResolve(SharedGraphBuilderPhase.java:158)
[WARN] [stderr]         ... 28 more
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.lookupMethod(BytecodeParser.java:4205)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genInvokeStatic(BytecodeParser.java:1648)
[WARN] [stderr] Caused by: java.lang.SecurityException: class "org.bouncycastle.pqc.jcajce.provider.util.KeyUtil"'s signer information does not match signer information of other classes in the same package
[WARN] [stderr]         at java.base/java.lang.ClassLoader.checkCerts(ClassLoader.java:1158)
[WARN] [stderr]         at java.base/java.net.URLClassLoader.defineClass(URLClassLoader.java:524)
[WARN] [stderr]         at java.base/java.lang.ClassLoader.preDefineClass(ClassLoader.java:902)
[WARN] [stderr]         at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1010)
[WARN] [stderr]         at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
[WARN] [stderr]         at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:427)
[WARN] [stderr]         at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:421)
[WARN] [stderr]         at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
[WARN] [stderr]         at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:420) at jdk.internal.vm.ci/jdk.vm.ci.hotspot.CompilerToVM.resolveTypeInPool(Native Method)
[WARN] [stderr] 
[WARN] [stderr]         at jdk.internal.vm.ci/jdk.vm.ci.hotspot.HotSpotConstantPool.loadReferencedType(HotSpotConstantPool.java:818)
[WARN] [stderr]         at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:587)
[WARN] [stderr]         at java.base/jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
[WARN] [stderr]         at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
[WARN] [stderr]         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[WARN] [stderr]         at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.loadReferencedType(WrappedConstantPool.java:152)
[WARN] [stderr]         ... 37 more
[WARN] [stderr] com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing org.bouncycastle.pqc.jcajce.provider.rainbow.BCRainbowPublicKey.getEncoded() 
[WARN] [stderr] Parsing context:
[WARN] [stderr]    at org.bouncycastle.pqc.jcajce.provider.rainbow.BCRainbowPublicKey.getEncoded(BCRainbowPublicKey.java:166)
[WARN] [stderr]    at sun.security.x509.X509Key.equals(X509Key.java:442)
[WARN] [stderr]    at java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1026)
[WARN] [stderr]    at java.util.concurrent.ConcurrentHashMap.putIfAbsent(ConcurrentHashMap.java:1541)
[WARN] [stderr] Fatal error:    at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:153)
[WARN] [stderr]    at com.oracle.svm.core.jdk.StringInternSupport.doIntern(StringInternSupport.java:77)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:104)
[WARN] [stderr]    at com.oracle.svm.core.jdk.StringInternSupport.intern(StringInternSupport.java:67)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
[WARN] [stderr]    at com.oracle.svm.core.jdk.Target_java_lang_String.intern(JavaLangSubstitutions.java:170)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
[WARN] [stderr] 
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultVirtualInvokeTypeFlow.onObservedUpdate(DefaultVirtualInvokeTypeFlow.java:109)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.TypeFlow.update(TypeFlow.java:562)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:488)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
[WARN] [stderr] Caused by: com.oracle.graal.pointsto.util.AnalysisError: parsing had failed in another thread
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.shouldNotReachHere(AnalysisError.java:177)
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:720)
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:171)
[WARN] [stderr]         at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:349)
[WARN] [stderr] Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Error loading a referenced type: java.lang.SecurityException: class "org.bouncycastle.pqc.jcajce.provider.util.KeyUtil"'s signer information does not match signer information of other classes in the same package
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
[WARN] [stderr]         at parsing org.bouncycastle.pqc.jcajce.provider.rainbow.BCRainbowPublicKey.getEncoded(BCRainbowPublicKey.java:169)
[WARN] [stderr]         ... 13 more
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2518)
[WARN] [stderr] 
[WARN] [stderr]         at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:110)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3393)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:97)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3345)
[WARN] [stderr]         at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:84)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3190)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)    ... 16 more
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:446)
[WARN] [stderr] Caused by: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Error loading a referenced type: java.lang.SecurityException: class "org.bouncycastle.pqc.jcajce.provider.util.KeyUtil"'s signer information does not match signer information of other classes in the same package
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.loadReferencedType(WrappedConstantPool.java:163)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.loadReferencedType(WrappedConstantPool.java:169)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
[WARN] [stderr]         at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SubstrateClassInitializationPlugin.loadReferencedType(SubstrateClassInitializationPlugin.java:58)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:685)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.loadReferenceType(BytecodeParser.java:4290)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.maybeEagerlyResolve(BytecodeParser.java:4272)
[WARN] [stderr]         at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.maybeEagerlyResolve(SharedGraphBuilderPhase.java:158)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.lookupMethod(BytecodeParser.java:4205)
[WARN] [stderr] Caused by: java.lang.SecurityException: class "org.bouncycastle.pqc.jcajce.provider.util.KeyUtil"'s signer information does not match signer information of other classes in the same package
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genInvokeStatic(BytecodeParser.java:1648)
[WARN] [stderr]         at java.base/java.lang.ClassLoader.checkCerts(ClassLoader.java:1158)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5288)
[WARN] [stderr]         at java.base/java.lang.ClassLoader.preDefineClass(ClassLoader.java:902)
[WARN] [stderr]         at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3385)
[WARN] [stderr]         at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1010)
[WARN] [stderr]         ... 28 more
[WARN] [stderr]         at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
[WARN] [stderr]         at java.base/java.net.URLClassLoader.defineClass(URLClassLoader.java:524)
[WARN] [stderr]         at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:427)
[WARN] [stderr]         at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:421)
[WARN] [stderr]         at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
[WARN] [stderr]         at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:420)
[WARN] [stderr]         at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:587)
[WARN] [stderr]         at jdk.internal.vm.ci/jdk.vm.ci.hotspot.CompilerToVM.resolveTypeInPool(Native Method)
[WARN] [stderr]         at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
[WARN] [stderr]         at jdk.internal.vm.ci/jdk.vm.ci.hotspot.HotSpotConstantPool.loadReferencedType(HotSpotConstantPool.java:818)
[WARN] [stderr]         at java.base/jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
[WARN] [stderr]         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[WARN] [stderr]         at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[WARN] [stderr]         at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.loadReferencedType(WrappedConstantPool.java:152)
[WARN] [stderr]         ... 37 more
```

--- a/integration-tests/pom.xml
+++ b/integration-tests/pom.xml
@@ -75,7 +75,7 @@
         <module>compression-grouped</module>
         <module>consul</module>
         <module>couchdb</module>
-        <!--<module>crypto</module> https://github.com/apache/camel-quarkus/issues/4510 -->
+        <module>crypto</module>
         <module>csimple</module>
         <module>csv</module>
         <!--<module>cxf-soap-grouped</module>-->

--- a/tooling/scripts/test-categories.yaml
+++ b/tooling/scripts/test-categories.yaml
@@ -108,6 +108,7 @@ group-07:
   - weather
 group-08:
   - azure-grouped
+  - crypto
   - master
   - master-file
   - master-openshift

